src/config/web3/evm/testnet/filecoin.ts

```diff
@@ -1,10 +1,10 @@
 import { ChainExtension } from "../interface";
 
 export const filecoin: ChainExtension = {
-  id: 3141,
+  id: 314159,
   name: "Filecoin EVM Testnet",
   network: "filecoin",
-  networkNameOverride: "filecoin",
+  networkNameOverride: "filecoin-2",
```

Comment:
Also need to change the explorer url to calibration instead of hyperspace testnet